go.mod

```diff
@@ -144,5 +144,5 @@ replace (
 	// Pin kubernetes hash
 	github.com/hashicorp/terraform-provider-kubernetes => github.com/hashicorp/terraform-provider-kubernetes v1.13.3-0.20201012211451-c24349628446
 	// Change kubevirt/terraform-provider-kubevirt to nirarg fork
-	github.com/kubevirt/terraform-provider-kubevirt => github.com/nirarg/terraform-provider-kubevirt v0.0.0-20201021155128-1c2b8ec764f0
+	github.com/kubevirt/terraform-provider-kubevirt => github.com/nirarg/terraform-provider-kubevirt v0.0.0-20201022163808-162e741630ed
```

Comment:
What is the reason for the change?
is it related to the hostname change?

Reply:
Removed the pod network from the VMs

Reply:
I wouldn't bundle it to the same PR/commit

Reply:
Its not the same commit, different commit for the modeules
I will squash all modules commits together, for the PR to openshift

Reply:
I don't think it should be at all in the same PR.
Here you are handling bootstrap and masters, workers are a different story.
also, for the bootstrap/masters I don't see that you removed the pods network interface

Reply:
Please notice: the Module I changed was `terraform-provider-kubevirt` which not related to the workers, also you can see the pods networks removal change in the following vendor file:
https://github.com/nirarg/installer/blob/7aa3accef327f4f84ed4d4183cff7af1269b517e/vendor/github.com/kubevirt/terraform-provider-kubevirt/kubevirt/virtualmachine/convert.go#L117

Reply:
IMO, different PR